### PR TITLE
LRAC-8177 Test fix IndividualDefinitions#AssertNewSourceInSourceListForIndividualAttribute

### PR DESCRIPTION
**Pull request by joshitagaki**
https://issues.liferay.com/browse/LRAC-8177

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase [hunk 1]
```diff
@@ -34,6 +34,8 @@ definition {
 	test AssertNewSourceInSourceListForIndividualAttribute {
 		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
 
+		var dataSource = StringUtil.replace("${propertyName}", " Combined Property", "");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase:37]: Good idea @joshitagaki !

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase [hunk 2]
```diff
@@ -45,8 +47,8 @@ definition {
 		ACDefinitions.goToDefinitionType(definitionType = "Individuals");
 
 		ACDefinitions.assertIndividualAttributesSource(
-			attributeList = "additionalName,address,birthDate,classPK,contactId,createDate,email,familyName,gender,givenName,jobTitle,modifiedDate,telephone,timeZoneId,userId,uuid",
-			dataSourceName = "Liferay DXP");
+			attributeList = "additionalName,address,birthDate,email,familyName,gender,givenName,jobTitle,telephone",
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase:50]: hey @joshitagaki , is it okay to not check these attributes?

**Review by alexcandidos — COMMENTED** [review]
Hey @joshitagaki , the code looks good to me! I just left a message regarding the attribute list.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase (hunk 2 above)
Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase:50]: Hey @alexcandidos, I'm not completely sure but I believe that some of those attributes I listed were never shown in the individual definitions as those are for more backend

**Comment by joshitagaki** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase (hunk 2 above)
Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase:50]: I see @joshitagaki ... Thanks for clarifying!

**Pull request closed without merging**